AsyncDisplayKit/ASTextNode.mm

```diff
@@ -355,6 +355,9 @@ - (void)setAttributedString:(NSAttributedString *)attributedString {
       self.isAccessibilityElement = YES;
     }
   });
+  
+  self.ascender = round([[attributedString attribute:NSFontAttributeName atIndex:0 effectiveRange:NULL] ascender] * [UIScreen mainScreen].scale)/[UIScreen mainScreen].scale;
```

Comment:
FYI, there is [ASScreenScale()](https://github.com/facebook/AsyncDisplayKit/blob/master/AsyncDisplayKit/Private/ASInternalHelpers.h#L19) that is already used in other places.